Kismet_CF_6.tex

```diff
@@ -0,0 +1,200 @@
+        \item Remote Fill Valve
+        \item Parallel Fill Valve
+        \item Motorized Vent Valve
+        \item Injector Valve
+    \end{checklist}
+    \item \primary{}: Confirm that the following valves are initially open:
+    \begin{checklist}
+	\item Series Fill Valve
+    \end{checklist}
+    \item \daq{}: Confirm that all pressure transducers are reading atmospheric pressure.
+    \item \daq{}: Confirm that all load cells are reading the determined zero point.
+    \item \textbf{\textit{PAUSE POINT}}
+    \item \peri{}, \perii{}, and \periii{}: Close the perimeter and do not allow any further personnel to enter the testing area.
+    \item \secondary: Confirm that no personnel are present in the testing area other than \primary{} and \secondary.
+    \item \primary{}: Remove the cap from the carbon dioxide supply cylinder.
+    \item \primary{}: Connect the fill line to the supply cylinder, hand tighten, and then tighten with a wrench. Do not force the connection.
+    \item \primary{}: Slowly open the Cylinder Valve through $\frac{3}{4}$  of a turn.
+    \begin{checklist}[label=$\bullet$]
+        \item If leaks are observed:
+        \begin{checklist}
+            \item \primary{}: Close the Cylinder Valve.
+            \item \primary{}: Slowly open the Parallel Fill Valve.
+            \item \daq{}: Confirm that the Fill Pressure Transducer is reading atmospheric pressure.
+            \item \ops{}: Abort test procedures and revisit plumbing setup.
+        \end{checklist}
+    \end{checklist}
+    \item \primary{}: Communicate the supply cylinder pressure as visible on the Pressure Gauge.
+    \item \daq{}: Communicate the supply cylinder pressure as read by the Fill Pressure Transducer.
+    \item \daq{}: Confirm that the two pressure measurements are in agreement.
+    \item \primary{} and \secondary{}: Retreat back to Mission Control.
+    \item \control{}: Perform the following control system checks:
+    \begin{checklist}
+        \item Confirm that all actuator controls are in the ``closed'' position:
+        \begin{checklist}
+            \item Remote Fill Valve
+            \item Motorized Vent Valve
+        \end{checklist}
+    \end{checklist}
+    \item \textbf{\textit{PAUSE POINT}}
+    \item \ops{}: Poll the following personnel for GO/NO GO status:
+    \begin{checklist}
+        \item \heat{}
+	\item \control{}
+        \item \daq{}
+        \item \primary{}
+        \item \secondary{}
+	\item \peri{}
+	\item \perii{}
+	\item \periii{}
+    \end{checklist}
+    \item \ops: Give the system control key to \control{}.
+    \item \control{}: Engage the key switch and power on the control boxes.
+    \item \control{}: Open the Motorized Vent Valve.
+    \item \control{}: Open the Remote Fill Valve.
+    \begin{checklist}[label=$\bullet$]
+        \item If leaks are observed:
+        \begin{checklist}
+            \item \control{}: Close the Remote Fill Valve.
+            \item \primary{}: Open the Injector Valve.
```

Comment:
I like to make this "Open the Injector Valve using the ropes" just to make it clear that they shouldn't need to walk up to the system